beacon-chain/rpc/eth/beacon/blocks_test.go

```diff
@@ -387,100 +417,105 @@ func TestServer_ListBlockHeaders(t *testing.T) {
 
 	_, blkContainers := fillDBTestBlocks(ctx, t, beaconDB)
 	headBlock := blkContainers[len(blkContainers)-1]
-	wsb, err := blocks.NewSignedBeaconBlock(headBlock.Block.(*ethpbalpha.BeaconBlockContainer_Phase0Block).Phase0Block)
-	require.NoError(t, err)
-	mockChainFetcher := &mock.ChainService{
-		DB:                  beaconDB,
-		Block:               wsb,
-		Root:                headBlock.BlockRoot,
-		FinalizedCheckPoint: &ethpbalpha.Checkpoint{Root: blkContainers[64].BlockRoot},
-	}
-	bs := &Server{
-		BeaconDB:              beaconDB,
-		ChainInfoFetcher:      mockChainFetcher,
-		HeadFetcher:           mockChainFetcher,
-		OptimisticModeFetcher: mockChainFetcher,
-	}
 
+	b1 := util.NewBeaconBlock()
+	b1.Block.Slot = 30
+	b1.Block.ParentRoot = bytesutil.PadTo([]byte{1}, 32)
+	util.SaveBlock(t, ctx, beaconDB, b1)
 	b2 := util.NewBeaconBlock()
 	b2.Block.Slot = 30
-	b2.Block.ParentRoot = bytesutil.PadTo([]byte{1}, 32)
+	b2.Block.ParentRoot = bytesutil.PadTo([]byte{4}, 32)
 	util.SaveBlock(t, ctx, beaconDB, b2)
 	b3 := util.NewBeaconBlock()
-	b3.Block.Slot = 30
-	b3.Block.ParentRoot = bytesutil.PadTo([]byte{4}, 32)
+	b3.Block.Slot = 31
+	b3.Block.ParentRoot = bytesutil.PadTo([]byte{1}, 32)
 	util.SaveBlock(t, ctx, beaconDB, b3)
 	b4 := util.NewBeaconBlock()
-	b4.Block.Slot = 31
+	b4.Block.Slot = 28
 	b4.Block.ParentRoot = bytesutil.PadTo([]byte{1}, 32)
 	util.SaveBlock(t, ctx, beaconDB, b4)
-	b5 := util.NewBeaconBlock()
-	b5.Block.Slot = 28
-	b5.Block.ParentRoot = bytesutil.PadTo([]byte{1}, 32)
-	util.SaveBlock(t, ctx, beaconDB, b5)
```

Comment:
This diff is misleading, I simply changed names of `b2/b3/b4/b5` to `b1/b2/b3/b4`.